src/content/keyAutoAdd/lib/keyAutoAdd.ts

```diff
@@ -87,6 +174,7 @@ export class KeyAutoAdd {
         }
         const details = this.errorToDetails(error);
         this.setStatus(stepIdx, 'error', { details: details });
+        await sleep(2000);
```

Comment:
We probably need to remove this?

Reply:
Not sure, I think I want user to see the step failing. Or maybe the redirect to welcome screen showing right error message be more than enough?

Reply:
If we add another value for the `success` parameter in the welcome screen I don't think we should need it at the moment. It depends on the UI - if we want the "loading bar". If we go for list, waiting for a couple of seconds would probably be okay.

Reply:
Ok, let's remove this for now. Can add again if needed.